core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackStopRestrictionService.java

```diff
@@ -22,7 +22,7 @@ public StopRestrictionReason isInfrastructureStoppable(String cloudPlatform, Set
         if (SUPPORTED_CLOUD_PLATFORM.equals(cloudPlatform)) {
             for (InstanceGroup instanceGroup : instanceGroups) {
                 if (instanceGroup.getTemplate().getVolumeTemplates().stream()
-                        .allMatch(volume -> AwsDiskType.Ephemeral.value().equals(volume.getVolumeType()))) {
+                        .anyMatch(volume -> AwsDiskType.Ephemeral.value().equals(volume.getVolumeType()))) {
                     reason = StopRestrictionReason.EPHEMERAL_VOLUMES;
                     LOGGER.info("Infrastructure cannot be stopped. Instances in group [{}] have ephemeral storage only.", instanceGroup.getGroupName());
```

Comment:
"Instances in group [{}] have ephemeral storage **_only_**."

Is this still true?